projects/laji/src/app/+project-form/results/biomon-result/biomon-result.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="container-fluid">
+  <laji-project-form-header [form]="form"></laji-project-form-header>
+  <div class="row">
+    <laji-navigation-thumbnail name="biomon.stats.statistics.title" icon="table" iconType="png"
```

Comment:
Sucks that the bs3 classes are kinda necessary here, because of the laji-navigation-thumbnail component. Maybe in the future that stuff could be refactored to use flexbox. Something like this should do the trick:
- container: `display: flex; flex-wrap: wrap; gap: ...px`
- items: `max-width: ...px`

But it's out of scope of this PR